build/constants.js

```diff
@@ -39,27 +39,17 @@ const FILES = process.env.BUILD_FILES || "";
 const FOLDERSEARCH = process.env.BUILD_FOLDERSEARCH || "";
 const GOOGLE_ANALYTICS_ACCOUNT =
   process.env.BUILD_GOOGLE_ANALYTICS_ACCOUNT || "";
-const GOOGLE_ANALYTICS_DEBUG = JSON.parse(
-  process.env.BUILD_GOOGLE_ANALYTICS_DEBUG || "false"
-);
+const GOOGLE_ANALYTICS_DEBUG = !!process.env.BUILD_GOOGLE_ANALYTICS_DEBUG;
```

Comment:
Sorry, I don't get what this PR does. If `export BUILD_GOOGLE_ANALYTICS_DEBUG=false` you'd want this variable to eventually be boolean `false`. 
But I do see now that if you use `export BUILD_GOOGLE_ANALYTICS_DEBUG=0` you'd end up with `GOOGLE_ANALYTICS_DEBUG` being an integer `0`.